### PR TITLE
[3.16] Add close_path parameter to points to path alg

### PR DESCRIPTION
**Pull request by DelazJ**
and fix output directory parameter name (backports part of #6220)

### CODE REVIEW

File: docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 1]
```diff
@@ -802,7 +809,7 @@ Outputs
      - [vector: line]
```
Comment (havatv) [inline docs/user_manual/processing_algs/qgis/vectorcreation.rst:809]:
```suggestion
     - [vector: line]
     
       Default: ``[Create temporary layer]``
```

File: docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 2]
```diff
@@ -802,7 +809,7 @@ Outputs
...
      - Line vector layer of the path
```
Comment (havatv) [inline docs/user_manual/processing_algs/qgis/vectorcreation.rst:810]:
```suggestion
     - Line vector layer of the path. One of:

       * Create Temporary Layer (``TEMPORARY_OUTPUT``)
       * Save to File...
       * Save to Geopackage...
       * Save to PostGIS Table...

       The file encoding can also be changed here.
```

**Review by havatv — APPROVED** [review]
A few suggestions.

File: docs/user_manual/processing_algs/qgis/vectorcreation.rst (hunk 2 above)
Comment (DelazJ) [reply docs/user_manual/processing_algs/qgis/vectorcreation.rst:810]: Not right. Here, you are in the output section and we do not repeat these details there afaik (see https://docs.qgis.org/3.10/en/docs/user_manual/processing_algs/qgis/vectorcreation.html#points-to-path)

**Pull request merged**